.github/workflows/release.yml

```diff
@@ -511,22 +511,73 @@ jobs:
 
   trigger-docs-update:
     if: ${{ inputs.skip-release-notes != true }}
-    needs: push-tags
+    needs:
+      - push-tags
+      - check-version
     runs-on: ubuntu-22.04
     timeout-minutes: 10
+    permissions:
+      contents: write
+      pull-requests: write
+    env:
+      NEW_OSS_VERSION: ${{ needs.check-version.outputs.oss }}
+      NEW_EE_VERSION: ${{ needs.check-version.outputs.ee }}
+      GH_TOKEN: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
     steps:
       - uses: actions/github-script@v7
         with:
-          github-token: ${{ secrets.DOCS_UPDATE_TOKEN }}
-          script: |
-            github.rest.repos.createDispatchEvent({
+          github-token: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
+          script: | # js
+            await github.rest.repos.createDispatchEvent({
```

Comment:
Curious if top level await works

Reply:
yep! actions script runs as [the body of an async function](https://github.com/actions/github-script#:~:text=the%20body%20of%20an%20asynchronous%20function%20call)